lib/top_score_filter.py

```diff
@@ -79,7 +80,9 @@ def execute(self):
                         else:
                             line_with_lowest_transcript_no = lines[0]
                             for line in lines:
-                                if float(line['Transcript'].replace('ENST','')) < float(line_with_lowest_transcript_no['Transcript'].replace('ENST','')):
+                                transcript_id = re.compile('ENST(\d+)').match(line['Transcript']).group(1)
+                                lowest_transcript_id = re.compile('ENST(\d+)').match(line_with_lowest_transcript_no['Transcript']).group(1)
```

Comment:
I'd be tempted to save the current lowest value instead of redetermining it on every loop iteration.  

(Do we still need the `float` casts that were in the previous version?  I'm not sure what they were protecting against here.)

Reply:
I wasn't sure whether an string comparison would yield different results then numerical comparisons (e.g.`'119' < '10'`) but I just tried it in a python console and it's doing the right thing without the float cast. 
I also made the change you suggested to save the current lowest value.